src/aspire/basis/pswf_2d.py

```diff
@@ -146,6 +146,8 @@ def _generate_samples(self):
         )
         self.samples = (self.beta / 2.0) * self.samples * self.alpha_nn
         self.samples_conj_transpose = self.samples.conj().transpose()
+        # the column dimension of samples_conj_transpose is the number of basis coefficients
+        self.count = self.samples_conj_transpose.shape[1]
```

Comment:
Relation to this PR?

Reply:
In order to be able to use the `UniversalBasisMixin` for all the basis tests, they all needed to have `self.count` defined. I believe we discussed adding this in a meeting

Reply:
Ok got it.